Library/Homebrew/readall.rb

```diff
@@ -77,9 +77,6 @@ def valid_tap?(tap, aliases: false, no_simulate: false, os_arch_combinations: ni
         success = false unless valid_formulae?(tap.formula_files)
         success = false unless valid_casks?(tap.cask_files)
       else
-        # TODO: Remove this default case once `--os` and `--arch` are passed explicitly to `brew readall` in CI.
```

Comment:
What CI does this refer to and has it been done?

Reply:
They were added in https://github.com/Homebrew/brew/pull/15470. The CI in this case is that used by other repos like homebrew-core and homebrew-cask (I assume those changes have already been made but I haven't checked to be sure). It cannot be removed without making some other small changes though (see https://github.com/Homebrew/brew/pull/15470#discussion_r1199672927). Beyond that maybe this can be cleaned up as well.

https://github.com/Homebrew/brew/blob/9b76182c119abca16eaa3b1faa64bd08ca112be3/Library/Homebrew/cmd/readall.rb#L54-L55

CC: @reitermarkus

Reply:
`brew test-bot` hasn't been updated.

>  Beyond that maybe this can be cleaned up as well.

Seems like that has to be cleaned up first, otherwise `os_arch_combinations` would be nil and would error.

Reply:
I'll resolve this before merging.

Reply:
I have re-added this TODO instead.